src/main/java/com/mongodb/hibernate/internal/translate/mongoast/filter/AstFilterOperation.java

```diff
@@ -18,4 +18,4 @@
 
 import com.mongodb.hibernate.internal.translate.mongoast.AstNode;
 
-interface AstFilterOperation extends AstNode {}
+public interface AstFilterOperation extends AstNode {}
```

Comment:
This interface does not need to be `public`.

```suggestion
interface AstFilterOperation extends AstNode {}
```